packages/mux-player/REFERENCE.md

```diff
@@ -15,6 +15,7 @@
 | `disable-cookies`                       | `boolean`                                                                                                   | Disables cookies used by Mux Data. For more, check out the [Mux Docs](https://docs.mux.com/guides/data/monitor-html5-video-element#disable-cookies).                                                                                                                                                                                                                                                         | `false`       |
 | `playback-id`                 | `string`                                                                                                    | The playback ID for your Mux Asset or Mux Live Stream. This will also be used for automatically assigning a [poster image](https://docs.mux.com/guides/video/get-images-from-a-video) and (thumbnail previews)[https://docs.mux.com/guides/video/create-timeline-hover-previews]. For more, check out the [Mux Docs](https://docs.mux.com/guides/video/play-your-videos#1-get-your-playback-id). | N/A           |
 | `prefer-playback`             | `"mse" \| "native"`                                                                                         | Specify if Mux Player should try to use Media Source Extension or native playback (if available). If no value is provided, Mux Player will choose based on what's deemed optimal for content and playback environment.                                                                                                                                                                           | Varies        |
+| `max-resolution`              | `"720p"                                                                                                      | Specify if Mux Player should try to use Media Source Extension or native playback (if available). If no value is provided, Mux Player will choose based on what's deemed optimal for content and playback environment.                                                                                                                                                                          | ""        |
```

Comment:
nitpick(blocking): `"720p"` should be `string`(progressive scan resolution) or similar. Also, description is inaccurate (copypasta).

Reply:
"progressive scan resolution" is too esoteric. Should just be "height of the video in pixels" or something like that. Ties into your suggestion that this should just be a number, too.

Reply:
> height of the video in pixels

This is tough because it's actually not height. A vertical video where the height is 1280 and the width is 720 would still fall under this. It's calculated by total pixels being`<= 921,600` which is 1280x720.

The only possible value we allow is `720p`, so I wouldn't want to give people the impression that they could specify any value they want.

I took at stab at updated, lmk your thoughts